models/user.js

```diff
@@ -0,0 +1,52 @@
+/* jshint indent: 1 */
+
+module.exports = function(sequelize, DataTypes) {
+	return sequelize.define('User', {
+		user_name: {
+			type: DataTypes.STRING(20),
+			allowNull: false
+		},
+		user_password: {
+			type: DataTypes.STRING(20),
+			allowNull: false
+		},
+		user_role_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: false,
+			references: {
+				model: 'UserRole',
+				key: 'id'
+			}
+		},
+		user_email_id: {
+			type: DataTypes.STRING(45),
+			allowNull: true
+		},
+		user_addr_1: {
```

Comment:
String length for address might be too short? How about 100 characters?

Reply:
Done.